Add docs for rolling content views (#3518)
theforeman/foreman-documentation: guides/common/modules/proc_creating-a-rolling-content-view.adoc

[id="creating-a-rolling-content-view"]
= Creating a rolling content view

> Use this procedure to create a rolling content view.

Reviewer: ```suggestion
You can create a rolling content view in the {ProjectWebUI}.
```
No self-referential content, please.

Author: Applied.

To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-rolling-content-view[].

.Procedure
. In the {ProjectWebUI}, navigate to *Content* > *Lifecycle* > *Content Views*.
. Click *Create content view*.
. In the *Create content view* window, enter a name for the content view in the *Name* field.
{Project} automatically completes the *Label* field from the name you enter.
. Optional: In the *Description* field, enter a description of the content view.
. On the *Type* tab, select *Rolling content view*.
. Click *Create content view*.

> . On the *Repositories* tab, select *All* from the *Status* list.

Reviewer 2: Doesn't it also have a "Show repositories" button in the empty state? That would be an easier flow.

Author: Yes it does. -> reworded.

. Select the repositories that you want to add to your rolling content view.
. Click *Add repositories* to add all selected repositories to your rolling content view.

[id="cli-creating-a-rolling-content-view"]
.CLI procedure

> . Before you create a rolling content views, list all available repositories:

Reviewer: ```suggestion
. List all available repositories to identify IDs of repositories to add to your rolling content view:
```
The order of steps is clear. No need for the "Before..." bit. Instead, let's tell them why they are doing it.

Author: Applied.

+
[options="nowrap" subs="+quotes"]
----
$ hammer repository list \
--fields id,name,product \
--organization "_My_Organization_"
----
. Create your rolling content view:
+
[options="nowrap" subs="+quotes"]
----
$ hammer content-view create \
--name "_My_Rolling_Content_View_" \
--organization "_My_Organization_" \
--repository-ids _My_List_Of_Repository_IDs_ \
--rolling
----

.Next steps

> * You cannot publish and promote your rolling content view.

Reviewer 2: ```suggestion
* You cannot publish or promote your rolling content view.
```

Author: Applied.

> Instead, continue with by adding it to your activation key.

Reviewer 2: ```suggestion
Instead, continue by adding it to your activation key.
```

Author: Thanks :see_no_evil: Applied.
